app/views/notes/_comment.html.erb

```diff
@@ -44,6 +44,9 @@
           <i class='icon fa fa-trash'></i>
         </a>
       <% end %>
+          <a rel="tooltip" title="Flag as spam." class="btn btn-sm btn-default" href="mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public Lab&body=Hi,+I+found+this+item+that+looks+like+spam+or+needs+to+be+moderated:+"<%= comment.body.replace(' ','+') %>"%0A%0Ahttps://publiclab.org/n/<%= node.id %>%0Aby+https://publiclab.org/profile/<%= comment.author.username %>%0A%0AThanks!">
```

Comment:
These lines look just right. I wonder if you could remove the others from your PR? 

https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

Then I think we'll be done here! Thanks so much!